cmd/flow-dps-indexer/main.go

```diff
@@ -66,6 +66,7 @@ func run() int {
 		flagIndexCommit       bool
 		flagIndexEvents       bool
 		flagIndexHeader       bool
+		flagIndexHeight       bool
```

Comment:
There is no real value to switching of indexing the height for each block, as it's almost entirely negligible for performance. On the other hand, it does not really make too much sense to index headers without also indexing their mappings. In short: this flag is unnecessary. Please just index the height as part of the header indexing (which also avoids the complicated optimization below).

Reply:
That was my first instinct, then I thought this would be more kosher. Bah. Will fix

Reply:
You know, it's OK to shoot us a message on Slack in cases like that :D